src/ex2/input.js

```diff
@@ -0,0 +1,53 @@
+class Input {
+  constructor() {
+    this.inputButton = null;
+  }
+
+  checkInputType(input) {
+    let newPokemonsIdArr = [];
+    let newNormalTasks = [];
+    if (isTextContainOnlyNumbersAndSpaces(input)) {
+      newPokemonsIdArr.push(input);
+    } else if (input.includes(",")) {
+      let arr = input.split(",");
+      newPokemonsIdArr = arr.filter(isTextContainOnlyNumbersAndSpaces);
```

Comment:
You can use also `parseInt`, `arr.filter(i => parseInt(i))`
using own written regex is usually not recommended as it might create security vulnerabilities